### ISSUE TITLE
Needs maintainers

### ISSUE THREAD

**Issue by gautamkrishnar** [gautamkrishnar/socli#111]
This project was started as one of my side projects. Now it has gone beyond my control. I am really thankful to the existing contributors and users of this project for making it a grand success 😄 

Due to my work schedules, I might be unavailable to do proper fixes for this project on time. I understand that the project is not at all manageable by a single person right now since it had grown further than I expected. I am **looking for some maintainers** for this project who can volunteer for the active development of this project. 

A maintainer *must have*:
* A good knowledge on python and PIP
* Knowlege on python requests, BeautifulSoup4, colorama, Py-StackExchange and urwid is required
* Experience in using Git and GitHub.
* Experience in using CI & CD like Travis CI

If you are interested in maintaining this project, please drop me an email with your profile and experience at r.gautamkrishna@gmail.com. Please mention any of your python projects and your involvement. Please comment here as well.

As a maintainer:
* You will be responsible for merging the pull requests without causing any unwanted effects to the existing features.
* You will be given write access to the development branch.
* You will be listed in the project's readme and CLI as a maintainer.

**Comment by gautamkrishnar** [comment]
Welcome @Aksh77 and @exepaul to the team :smile:

**Comment by aashutoshrathi** [comment]
Hey, @gautamkrishnar I would like to help in maintaining this.
I will go through the codebase in 2 days, I think. Also will send some PRs.

**Comment by gautamkrishnar** [comment]
@aashutoshrathi Cool! 😄Really happy to have you here. This requires some serious merging of PRs and testing the code. Granted you push access to the repo. https://github.com/gautamkrishnar/socli/invitations

Please make sure that you work on the develop branch while merging PRs.

**Comment by aashutoshrathi** [comment]
I'll go through the code base and start checking PRs. Thanks
As of now, I think it is not responding to queries.

**Comment by gautamkrishnar** [comment]
@aashutoshrathi yes it has been a long time since it is updated. Have you tried installing the latest version from Pypi:
```
sudo pypi install socli
```

**Comment by gautamkrishnar** [comment]
Looks like it is not working as well...

**Comment by aashutoshrathi** [comment]
Yes.

**Comment by Saamuuu** [comment]
Hello @gautamkrishnar, I'm also interested in maintaining the repo as well. I'll be sure to test out the software to see if its working. Then check out the issues in the repo and/or address pull requests.

**Comment by gautamkrishnar** [comment]
Re-opening this since we need more maintainers. Feel free to comment on this issue if you are interested in maintaining the project.
